docs/agents/latrt_xy_stage.rst

```diff
@@ -0,0 +1,72 @@
+.. highlight:: rst
+
+.. _latrt_xy_stage:
+
+=====================
+LATRt XY Stage Agent
+=====================
+
+This agent is used to communicate with the XY Stages used in the LATRt lab.
+These stages are run off a Raspberry Pi connected to some custom electronics
+boards for communicating with the stages. 
+
+Since control of these stages need to be accessible inside and outside OCS,
+their drivers are shared `here
+<https://github.com/kmharrington/xy_stage_control>`_. 
+
+.. argparse::
+    :filename: ../agents/xy_stage/xy_latrt_agent.py
+    :func: make_parser
+    :prog: python3 xy_latrt_agent.py
+
+
+Configuration File Examples
+---------------------------
+Below are configuration examples for the ocs config file and for running the
+Agent in a docker container.
+
+ocs-config
+``````````
+To configure the LATRt XY Stage Agent we need to add a block to our ocs 
+configuration file. Here is an example configuration block using all of 
+the available arguments::
+
+      {'agent-class': 'LATRt_XY_StageAgent',
```

Comment:
Needs underscores removed, and this needs to match the `agent_class` passed to `site_config.parse_args()`, so it should be `'LATRtXYAgent'`.